pkgs/applications/graphics/krita/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, cmake, kdelibs, attica, perl, zlib, libpng, boost, mesa
+, kdepimlibs, createresources ? null, eigen, qca2, exiv2, soprano, marble, lcms2
+, fontconfig, freetype, sqlite, icu, libwpd, libwpg, pkgconfig, poppler_qt4
+, libkdcraw, libxslt, fftw, glew, gsl, shared_desktop_ontologies, okular
+, libvisio, kactivities, mysql, postgresql, freetds, xbase, openexr, ilmbase
+, libodfgen, opencolorio, openjpeg, pstoedit, librevenge, oxygen_icons
+ }:
+
+stdenv.mkDerivation rec {
+  name = "krita-2.9.5";
```

Comment:
Heh, the version is wrong :-) Better not repeat the version &ndash; it will avoid such problems in future.